openmoc/thisown.i

```diff
@@ -107,6 +107,23 @@
         universes[i][j][k].thisown = False
 %}
 
+/* A Lattice owns the memory for each Universe it contains */
+%pythonappend Lattice::updateUniverse %{
+  # SWIG 3
+  if 'universe' in locals():
+    universe = locals()['universe']
```

Comment:
This line should extract the `universe` variable from the list of local variables. Basically, it's a way of getting access to the parameter called `locals` for the `Lattice::updateUniverse(...)` method.